src/main/resources/com/owlcyberdefense/mil-std-2045/xsd/milstd2045.enums.dfdl.xsd

```diff
@@ -92,6 +92,12 @@ SOFTWARE.
     <xs:restriction base="ms2045:enumBase"/>
   </simpleType>
 
+  <simpleType name="messageVersion">
+    <xs:restriction base="xs:string">
+      <xs:maxLength value="13"/>
```

Comment:
These changes to the actual schema should go on the master/main branch. 

It's just stuff specific to xsat2 testing that we want kept off the main branch.